index.js

```diff
@@ -29,102 +35,65 @@ app.use(session({
 
 app.use(bodyParser.urlencoded({ extended: false}));
 
-app.use(methodOverride(function (req, res) {
-  if (req.body && typeof req.body === 'object' && '_method' in req.body) {
-    // look in urlencoded POST bodies and delete it
-    var method = req.body._method;
-    delete req.body._method;
-    return method;
-  }
-}));
-
 app.use('/admin', adminRouter);
+app.use('/authentication', authenticationRouter);
 
-// comment posted to db
-app.post('/posts/:id/comments', (req, res) => {
-  db.Post.findById(req.params.id).then((post) => {
-    var comment = req.body;
-    comment.PostId = post.id;
-    db.Comment.create(comment).then(() => {
-      res.redirect('/' + post.slug);
-        });
+
+
+//gets home page
```

Comment:
your code is self documenting, no need for comments like this. Only use comment for something very complex where you really have to use a comment however even that is troubling most of the time.